src/dodal/devices/shutter.py

```diff
@@ -0,0 +1,47 @@
+from enum import Enum
+
+from bluesky.protocols import Movable
+from ophyd_async.core import AsyncStatus, StandardReadable, wait_for_value
+from ophyd_async.epics.signal import epics_signal_r, epics_signal_w
+
+
+class ShutterSetState(str, Enum):
+    CLOSE = "Close"
+    OPEN = "Open"
+    RESET = "Reset"
+
+
+class ShutterState(str, Enum):
+    CLOSED = "Closed"
+    OPEN = "Open"
+    OPENING = "Opening"
+    CLOSING = "Closing"
+    FAULT = "Fault"
```

Comment:
```suggestion
```suggestion
class ZebraShutterState(str, Enum):
    CLOSE = "Close"
    OPEN = "Open"
```
```

Reply:
added